data_processing/pathology/common/preprocess.py

```diff
@@ -21,9 +23,32 @@
 from skimage.filters import threshold_otsu
 from skimage.draw import rectangle_perimeter, rectangle
 
-NUM_COLORS = 100 + 1
-scoring_palette = sns.color_palette("viridis_r", n_colors=NUM_COLORS)
-scoring_palette_as_list = [[int(x * 255) for x in scoring_palette.pop()] for i in range(NUM_COLORS)]
+from urllib.request import urlopen
+from shapely.geometry import shape, Point, Polygon
+
+from random import randint
+
+
+palette = sns.color_palette("viridis",as_cmap=True)
+categorical_colors = {}
+def get_tile_color(score):
```

Comment:
Nice function!!!  There's a huge backlog of unit-tests, this is a good candidate too. I'll add these to the backlog ~